Cargo.toml

```diff
@@ -15,6 +15,8 @@ include = ["src/**/*", "LICENSE", "README.md"]
 [build-dependencies]
 clap = "3"
 clap_complete = "3"
+winres = "0.1"
+chrono = "0.4"
```

Comment:
Can those 2 be added with `[target.'cfg(windows)'.build-dependencies]` or whatever would be the right TOML key to only require those when building on Windows?

Reply:
> Can those 2 be added with `[target.'cfg(windows)'.build-dependencies]` or whatever would be the right TOML key to only require those when building on Windows?

No, due to issues in `cargo`, this is currently not possible and will break `build.rs` build script. See https://github.com/rust-lang/cargo/issues/4932 for more information.

Reply:
oh ok :(
Can we use `time` instead of chrono?

Reply:
> oh ok :( Can we use `time` instead of chrono?

Sure thing. Which version of `time` do you want me to use? `0.1` series or the latest `0.3` series?

Reply:
Rebased and switched to `time 0.3`.